recipes/datarobot-bp-workshop/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "datarobot-bp-workshop" %}
+{% set version = "0.2.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/datarobot-bp-workshop-{{ version }}.tar.gz
+  sha256: 0f5e94206d89ea73d5798c88e145e3b2d8ebfadb258f30ca6374259813257582
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - pytest
+    - python >=3.7
+  run:
```

Comment:
It looks like you are listing many more dependencies than are in the `setup.py`, which only specifies:

```
"datarobot>=2.27.0,<3"
"networkx>=2.2"
"graphviz>=0.16,<1"
```

Typically you should only need those plus `python`. I looked in the commits and it looks like you're listing the `datarobot` dependencies, but those should be pulled in `datarobot`:
https://github.com/conda-forge/datarobot-feedstock/blob/main/recipe/meta.yaml#L23-L36

and don't need to be included here.

Reply:
I've removed the unneeded dependencies